alvr/client_openxr/src/stream.rs

```diff
@@ -419,7 +426,12 @@ impl StreamContext {
                             .image_rect(rect),
                     ),
             ],
-            false,
+            self.config
+                .passthrough
+                .clone()
+                .map(|mode| ProjectionLayerAlphaConfig {
+                    premultiplied: !matches!(mode, PassthroughMode::Blend { .. }),
```

Comment:
Do you want to add additional options or why not just `matches!(mode, PassthroughMode:AugmentedReality`
Because `!matches!` is kinda cursed

Reply:
Yes, there will be also a ChromaKey option. Chroma Kye will not have a brightness/opacity slider so using another variant will hide that.

Reply:
Also, as you saw, opacity and brightness has the same meaning in code. I made them separate options to have clearer names in the dashboard.

Reply:
I would've said to prefer `.not()` in this case then, but it doesn't really matter

Reply:
I didn't know it was a thing lol. But also it's kinda obscure and less visible IMO

Reply:
Maybe instead I should match `AugmentedReality {} | ChromaKey {}`